recipes/gnuradio-symbolmapping/meta.yaml

```
@@ -0,0 +1,65 @@
+{% set oot_name = "symbolmapping" %}
+{% set name = "gnuradio-" + oot_name %}
+{% set version = "1.0.3" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/ant-uni-bremen/gr-symbolmapping/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: 4a1002d54ab70dfd0616aa6d3eb1a1a27cf8a60d1445b47b0c9643eb3173cbaf
+
+build:
+  skip: True  # [win]
+  number: 0
+
+requirements:
+  build:
+    - {{ compiler("c") }}
+    - {{ compiler("cxx") }}
+    - cmake >=3.16
+    - git
+    - ninja
+    - pkg-config  # [not win]
+    # cross-compilation requirements
+    - python                              # [build_platform != target_platform]
+    - cross-python_{{ target_platform }}  # [build_platform != target_platform]
+    - numpy                               # [build_platform != target_platform]
+    - pybind11                            # [build_platform != target_platform]
+    # Add extra build tool dependencies here
+
+  host:
+    - python
+    - numpy >=1.22
+    - fmt
+    - spdlog
+    - pybind11
+    - gmp  # [not win]
+    - fftw
+    - volk >=2.5
+    - gmp  # [linux]
+    # the following two entries are for generating builds against specific GR versions
+    - gnuradio-core  # [not gnuradio_extra_pin]
+    - gnuradio-core {{ gnuradio_extra_pin }}.*  # [gnuradio_extra_pin]
```

Don't use the `[]` when you actually are not using the functionality.
```suggestion
    - gnuradio-core  # not gnuradio_extra_pin
    - gnuradio-core {{ gnuradio_extra_pin }}.*  # gnuradio_extra_pin
```

---

I suspect either this change, or the `- gmp  # [unix]` change broke the build. At least it seems to work again after i reverted those changes.

---

I've tried all kinds of different combinations. The original version is the only one that works. Also, the original version basically expresses an XOR between the two. This should be the correct behavior.

---

Where does the `gnuradio_extra_pin` variable get defined? You haven't included a 'conda_build_config.yaml', so `gnuradio_extra_pin` should have no value.

`gnuradio-core` is [pinned globally on conda-forge](https://github.com/conda-forge/conda-forge-pinning-feedstock/blob/5bac11c21f2a14b9100b9127848310df7721b25f/recipe/conda_build_config.yaml#L347) to the minor version, so ideally we want you to not pin it again.

---

I pulled these lines from the GNU Radio OOT template.
If I get this correctly, the way forward would be to just have one line
```yaml
- gnuradio-core
```
without anything extra. Would that be correct?